dev/plugins/hu.elte.txtuml.export.javascript/src/hu/elte/txtuml/export/javascript/resources/res/pollingRefresh.js

```diff
@@ -88,3 +89,34 @@ $('#debug-port-input').on('keyup change input', function(){
 		typingTimer = setTimeout(doneTyping, DONE_TYPING_TIMEOUT_IN_MILLISECONDS);
 	}
 });
+
+//send animation delay to the server
+function sendSMTime(){
+	$.ajax({
+		url: 'http://localhost:' + port + '/delay',
+		type: 'POST',
+		dataType: 'json',
+		data: {delayTime:stateMachineSpeed},
```

Comment:
The _delay_ and _speed_ terminology should be made consistent throughout the implementation. I suggest _delay_.